gcassert_test.go

```diff
@@ -127,11 +127,11 @@ testdata/issue5.go:4:	Gen().Layout(): call was not inlined
 		{
 			name: "relative-cwd",
 			pkgs: []string{
-				".",
-				"./otherpkg",
+				"./testdata",
+				"./testdata/otherpkg",
 			},
-			cwd: filepath.Join(cwd, "testdata"),
-			expected: strings.ReplaceAll(expectedOutput, "testdata/", ""),
+			cwd: cwd,
+			expected: expectedOutput,
```

Comment:
I changed the `cwd` used here from `testdata` to the root directory as I was getting failures:

```
            	            	Diff:
            	            	--- Expected
            	            	+++ Actual
            	            	@@ -1,12 +1,3 @@
            	            	-noescape.go:21:	foo := foo{a: 1, b: 2}: foo escapes to heap:
            	            	-noescape.go:35:	// This annotation should fail, because f will escape to the heap.
            	            	-//
            	            	-//gcassert:noescape
            	            	-func (f foo) setA(a int) *foo {
            	            	-	f.a = a
            	            	-	return &f
            	            	-}: f escapes to heap:
            	            	-noescape.go:45:	: a escapes to heap:
            	            	-bce.go:8:	fmt.Println(ints[5]): Found IsInBounds
            	            	-bce.go:23:	fmt.Println(ints[1:7]): Found IsSliceInBounds
            	            	+bce.go:11:	sum += inlinable(ints[i]): call was not inlined
            	            	+bce.go:13:	sum += inlinable(ints[i]): call was not inlined
            	            	 bce.go:17:	sum += notInlinable(ints[i]): call was not inlined
            	            	@@ -14,4 +5,6 @@
            	            	 inline.go:45:	alwaysInlined(3): call was not inlined
            	            	+inline.go:49:	sum += inlinable(i): call was not inlined
            	            	 inline.go:51:	sum += notInlinable(i): call was not inlined
            	            	 inline.go:55:	sum += 1: call was not inlined
            	            	+inline.go:57:	test(0).alwaysInlinedMethod(): call was not inlined
            	            	 inline.go:58:	test(0).neverInlinedMethod(10): call was not inlined
```

I don't know what's going on with this, but it seems like the `go` compiler is giving us different output depending on which `cwd` you choose. 🤷‍♂️